server/app/controllers/ti/TrustedIntermediaryController.java

```diff
@@ -114,11 +167,15 @@ public Result addApplicant(Long id, Http.Request request) {
     if (Strings.isNullOrEmpty(form.get().getLastName())) {
       return redirectToDashboardWithError("Last name required.", form);
     }
+    if (Strings.isNullOrEmpty(form.get().getDob())) {
```

Comment:
issue (blocking): please validate the DOB for:

being in the correct format
that the date is in the past

Reply:
@swatkat1 this comment was resolved without having been addressed